ipatests/test_integration/test_ntp_options.py

```diff
@@ -47,15 +34,18 @@ def test_server_client_install_without_options(self):
                         "no NTP server or pool address was provided."
         expected_msg2 = "Using default chrony configuration."
 
-        server_install = tasks.install_master(self.master, setup_dns=False)
-        assert expected_msg1 in server_install.stderr_text
-        assert expected_msg2 in server_install.stdout_text
+        try:
+            server_install = tasks.install_master(self.master, setup_dns=False)
+            assert expected_msg1 in server_install.stderr_text
+            assert expected_msg2 in server_install.stdout_text
 
-        client_install = self.install_client()
-        assert expected_msg1 in client_install.stderr_text
-        assert expected_msg2 in client_install.stdout_text
+            client_install = tasks.install_client(self.master, self.client)
 
-        self.cleanup()
+            assert expected_msg1 in client_install.stderr_text
+            assert expected_msg2 in client_install.stdout_text
+
+        finally:
```

Comment:
Please replace ``try/finally: self.cleanup`` with ``teardown_method`` hook: https://docs.pytest.org/en/latest/xunit_setup.html#method-and-function-level-setup-teardown